ui/src/components/Dashboard/index.jsx

```diff
@@ -54,7 +60,12 @@ export default function Dashboard(props) {
       <Helmet>
         <title>{title} - Balrog Admin</title>
       </Helmet>
-      <AppBar className={classes.appbar}>
+      <AppBar
+        className={`${classes.appbar} ${
+          process.env.NODE_ENV === 'development' ? classes.local : null
```

Comment:
Mm is `process.env.NODE_ENV` available in the production build?

I think we'd have to do something like to make it available https://github.com/neutrinojs/neutrino/issues/82#issuecomment-417226109

Reply:
Mm maybe `process.env` is available, looks like we reference it in a few places like https://github.com/mozilla-releng/balrog/blob/05036afc2db59837938aae4423ec81e7eaa57b93/ui/src/utils/constants.js#L2

Reply:
Do we need to add it here? https://github.com/mozilla-releng/balrog/blob/05036afc2db59837938aae4423ec81e7eaa57b93/ui/.neutrinorc.js#L64-L76

Reply:
> Mm is `process.env.NODE_ENV` available in the production build?
> 
> I think we'd have to do something like to make it available [neutrinojs/neutrino#82 (comment)](https://github.com/neutrinojs/neutrino/issues/82#issuecomment-417226109)

Thank you for updating me. The `process.env.NODE_ENV` would not be available in the production build. I will be using the DefinePlugin plugin to create a constant for it.

Reply:
Do you need some help adding it? Once you add it I can land this to try it out in the staging instance :)